tests/chainer_tests/links_tests/loss_tests/test_crf1d.py

```diff
@@ -0,0 +1,69 @@
+import unittest
+
+import numpy
+
+import chainer
+from chainer.backends import cuda
+from chainer import initializers
+from chainer import links
+from chainer import testing
+from chainer.testing import attr
+
+
+@testing.parameterize(*testing.product({
+    'dtype': [numpy.float16, numpy.float32, numpy.float64]
+}))
+class TestCRF1d(unittest.TestCase):
+
+    n_labels = 3
+
+    def setUp(self):
+        self._config_user = chainer.using_config('dtype', self.dtype)
+        self._config_user.__enter__()
```

Comment:
You don't need to change the config. Actually I think it's okay to test a single dtype, e.g. float32, without parameterizing over it.

Reply:
https://github.com/chainer/chainer/pull/5807#discussion_r241462351

Reply:
Sorry but was this comment addressed?

Reply:
So is it OK not to do anything?

Reply:
I'd say that.